### DISCUSSION TITLE
JavaScript syntax validation: duplicate variable or object member hint

### DISCUSSION THREAD

**Discussion by chenjie100** [microsoft/monaco-editor#3739] [Q&A] [answered]
consider these code:
let foo, bar, foo;
let object = {foo: 123, bar: 'abc', foo: 456} //semicolon missing 
monaco does not give any "Duplicate variable or member hint", "Semicolon missing hint"

**Comment by hediet** [comment, marked as answer]
https://microsoft.github.io/monaco-editor/playground.html?source=v0.36.1#XQAAAAJoAgAAAAAAAABBqQkHQ5NjdMjwa-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhaLFIw9vEWSxW3yscw10ok-_1uhsmGN6AJu7JrrwwtX9cVOVfDKV-jI2oGSQUu4Hj8aI7SDUASS6-begmAD03IGW_3sPOFqEWQJTyXQm__kvfk7qqWsJvje-NcjbiUd5ltryQKZc_D3MkpyxnG4-qxtcTj632p3OewAI37xs1imCTUPmBA1Dd97EEOAh0OrSy-AuJdW_lojjzvCDYDrTvWJpt1YAdwNrW3S1gwzXJz7VLncwsn9WLNUty4q4JhCawCX5lF2eHkUP3X0YATTjRnHY6DtCyfKIE7F_eAuMx3cdBMlL8RqLKf5bCW-rLs4_a8F78pGlHSogbU8PiHR4KmxKSD_LY-oPxku5LgHMfwM7ebjl3qSEzPxsbeT35LS-dmaaH9-3ozk2P6crwQBv19mZZDrYuBZ7o0dopMXlixrVrp__Mu8c4
